Content.Shared/CCVar/CCVars.cs

```diff
@@ -557,6 +557,24 @@ public sealed class CCVars : CVars
         public static readonly CVarDef<float> ShuttleDockSpeedCap =
             CVarDef.Create("shuttle.dock_speed_cap", 5f, CVar.SERVERONLY);
 
+        public static readonly CVarDef<float> ShuttleMaxLinearAcc =
+            CVarDef.Create("shuttle.max_linear_acc", 13f, CVar.SERVERONLY);
+
+        public static readonly CVarDef<float> ShuttleMaxAngularAcc =
+            CVarDef.Create("shuttle.max_angular_acc", 3f, CVar.SERVERONLY);
```

Comment:
Call it Impulse instead of Acc

Reply:
I mean it's not really a cap on impulse, because impulse depends on mass.